config/nginx.conf

```diff
@@ -25,3 +25,69 @@ server {
         client_max_body_size 25M;
     }
 }
+server {
+    listen 8044 default_server;
+    index index.html
+
+    # To allow special characters in headers
+    ignore_invalid_headers off;
+    # Allow any size file to be uploaded.
+    # Set to a value such as 1000m; to restrict file size to a specific value
+    client_max_body_size 0;
+    # To disable buffering
+    proxy_buffering off;
+
+    location / {
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_set_header Host $http_host;
+
+        proxy_redirect off;
+        proxy_connect_timeout 300;
+        # Default is HTTP/1, keepalive is only enabled in HTTP/1.1
+        proxy_http_version 1.1;
+        proxy_set_header Connection "";
+        chunked_transfer_encoding off;
+
+        # If our URI doesn't contain a period and also doesn't have a slash at the end, add one
+        rewrite ^([^.]*[^/])$ $1/ permanent;
+        # If our URI ends with a slash, add index.html to the end
+        rewrite /$ ${request_uri}index.html last;
+        # Pass to the Minio preview bucket
+        proxy_pass http://s3:9000/${AWS_PREVIEW_BUCKET_NAME}/;
+    }
+}
+server {
+    listen 8045 default_server;
+    index index.html
+
+    # To allow special characters in headers
+    ignore_invalid_headers off;
+    # Allow any size file to be uploaded.
+    # Set to a value such as 1000m; to restrict file size to a specific value
+    client_max_body_size 0;
+    # To disable buffering
+    proxy_buffering off;
+
+    location / {
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_set_header Host $http_host;
+
+        proxy_redirect off;
+        proxy_connect_timeout 300;
+        # Default is HTTP/1, keepalive is only enabled in HTTP/1.1
+        proxy_http_version 1.1;
+        proxy_set_header Connection "";
+        chunked_transfer_encoding off;
+
+        # If our URI doesn't contain a period and also doesn't have a slash at the end, add one
+        rewrite ^([^.]*[^/])$ $1/ permanent;
+        # If our URI ends with a slash, add index.html to the end
+        rewrite /$ ${request_uri}index.html last;
+        # Pass to the Minio publish bucket
+        proxy_pass http://s3:9000/${AWS_PUBLISH_BUCKET_NAME}/;
+    }
+}
```

Comment:
@mbertrand The nginx.conf file is only used in dev I believe.  The comment at the top says "This is the version used in development environments."  It's pulled in here in `docker-compose.yml`:

```
  nginx:
    volumes:
      - ./config/nginx.conf:/etc/nginx/templates/web.conf.template
      - ./:/src
```

So, I believe that Heroku doesn't use this file at all, but if you know otherwise definitely let me know.